engine/combat/utility/core/action.py

```diff
@@ -0,0 +1,17 @@
+# When calculating an action to pick, the reasoner will select the
+# an action and return the consideration and appraisal as an Action
+# which will be consumed to execute on an action.
+# The consideration will execute to setup for the action, and
+# then the appraisal will execute.
+
+
+class Action:
+    def __init__(self, consideration, appraisal):
+        self.consideration = consideration
+        self.appraisal = appraisal
+
+    def execute(self):
```

Comment:
Is this used at all? You seem to bypass it in the `CombatController` by calling these functions directly.

Reply:
It is not used :P there there sort of as a placeholder atm, i'd like to abstract this away a bit more so it can be done here, though.